tests/testthat.R

```diff
@@ -1,2 +0,0 @@
-library(testthat)
-test_check("ROhdsiWebApi")
```

Comment:
Isn't this file required per https://r-pkgs.org/tests.html#test-workflow or perhaps there is a newer approach?

Reply:
I took this pattern from packages that use DatabaseConnector on different platforms. Ideally we would have a WebApi test instance that supports Db, active directory windows etc login platforms so the files `testAdAuth.R testWindowAuth.R, testDbAuth.R` represent moving towards that goal... but I can revert this if you think an alternative pattern is better.

Reply:
Fair enough - if what you've done allows for unit testing, I'm good with this change.

Reply:
@anthonysena do you approve the changes. Are we ok to merge?

Reply:
Yes, just left the approval. Thanks!